lib/whatsapp_sdk/api/messages.rb

```
@@ -112,8 +121,10 @@ def send_image(
       # @param audio_id [String] Audio ID.
       # @param link [String] Audio link.
       # @param message_id [String] The id of the message to reply to.
+      # @param voice [Boolean] Whether the audio is a voice note.
+      # @param recipient [String] Recipient' Business-Scoped User ID (BSUID). Optional alternative to recipient_number.
       # @return [MessageDataResponse] Response object.
-      def send_audio(sender_id:, recipient_number:, audio_id: nil, link: nil, message_id: nil, voice: false)
+      def send_audio(sender_id:, recipient_number: nil, audio_id: nil, link: nil, message_id: nil, voice: false, recipient: nil)
```

RuboCop (1.87.0)

[convention] Line is too long. [128/120]

(Layout/LineLength)